openlibrary/components/IdentifiersInput.vue

```diff
@@ -60,6 +60,8 @@ const identifierPatterns  = {
     amazon: /^B[0-9A-Za-z]{9}$/,
     youtube: /^@[A-Za-z0-9_\-.]{3,30}/,
 }
+const popularEditionIds = ['ocaid', 'isbn_10', 'isbn_13', 'lccn', 'librivox', 'oclc_numbers', 'project_gutenberg']
```

Comment:
This would be better as a property that can be specified in the HTML that creates this component; that more cleanly defines the boundary between the vue component and the outside world. This component shouldn't know "how" it's being used ; it should just know about the public interface it exposes -- and that's (popular_ids, multiple, config_file)